lte/gateway/c/core/oai/test/mme_app_task/test_mme_procedures_x2_handover.cpp

```diff
@@ -122,4 +122,4 @@ TEST_F(MmeAppProcedureTest, TestX2HandoverPathSwitchSuccess) {
 }
 
 }  // namespace lte
-}  // namespace magma
```

Comment:
I think new line is not required

Reply:
There is no new line. I think its because of formatting